src/storages/pluggable/inMemoryWrapper.ts

```diff
@@ -6,10 +6,13 @@ import { ISet, setToArray, _Set } from '../../utils/lang/sets';
  * Creates a ICustomStorageWrapper implementation that stores items in memory.
  * The `_cache` property is the object were items are stored.
  * Intended for testing purposes.
+ *
+ * @param connDelay delay in millis for `connect` resolve. If not provided, `connect` resolves inmediatelly.
  */
-export function inMemoryWrapperFactory(): ICustomStorageWrapper & { _cache: Record<string, string | string[] | ISet<string>> } {
+export function inMemoryWrapperFactory(connDelay?: number): ICustomStorageWrapper & { _cache: Record<string, string | string[] | ISet<string>>, _setConnDelay(connDelay: number): void } {
```

Comment:
Is this parameter only used in testing?

Reply:
yes Nico. At the moment, `inMemoryWrapperFactory` is only used in tests, but t is not inside a `__tests__` folder, because those folders are not part of the uploaded NPM package, and so cannot be reused in other packages.